core/src/main/java/com/amplifyframework/core/Amplify.java

```diff
@@ -142,9 +135,7 @@ public static void configure(@NonNull final AmplifyConfiguration configuration,
                 }
             }
 
-            if (debuggableApp) {
-                DeveloperMenuManager.sharedInstance(context).startListening();
-            }
+            DeveloperMenuManager.sharedInstance(context).enableDeveloperMenu();
```

Comment:
This is way cleaner, thanks!

I'd ditch the `Manager` part, you could just call it a `DeveloperMenu`. (Ref: https://blog.codinghorror.com/i-shall-call-it-somethingmanager/)

The `sharedInstance` verbiage is idiomatic to the iOS platform, but not to Android. On Android, this would probably be expressed as just `instance` (if you can have multiple) or `singleton`, or `singletonInstance`, if the code internally prevents multiple instances from being created.